### DISCUSSION TITLE
如果要是设置密码或者是隐藏目录必须的一个一个设置吗

### DISCUSSION THREAD

**Discussion by kkule** [AlistGo/alist#577] [Q&A]
如果要是设置密码或者是隐藏目录必须的一个一个设置吗
还有就是隐藏的目录是连下面的文件都隐藏了吗

**Comment by xhofe** [comment]
密码可以只给根目录设置，隐藏需要一个一个设置，前端隐藏可以使用正则。
隐藏目录只是页面上不显示，可以手动输入URL进入。

**Reply by cvpas** [reply]
> 密码可以只给根目录设置，隐藏需要一个一个设置，前端隐藏可以使用正则。 隐藏目录只是页面上不显示，可以手动输入URL进入。

请问有什么办法可以让隐藏的文件无法通过输入URL访问呢？
